article/article.txt

```diff
@@ -131,11 +131,24 @@ How to access the John Hopkins University Covid-19 Data
    If you would prefer to host the data yourself, the scripts required to download and transform the JHU data are open-source. You can `view them and instructions for how to use them on our GitHub repository <https://github.com/mongodb-developer/open-data-covid-19/tree/master/data-import>`_. 
    
 
-   Demos
-   -------------------------
+   Get Started
+   -----------
+
+   You can begin exploring the data right away without any MongoDB or programming experience using `Charts <https://www.mongodb.com/products/charts>`_ or `Compass <https://www.mongodb.com/products/compass>`_.
+
+   With `Charts <https://www.mongodb.com/products/charts>`_, you can create visualisations of the data using any of the pre-built graphs and charts. You can then `arrange this into a unique dashboard <https://charts.mongodb.com/charts-coronavirus-lwlvn/public/dashboards/5e7a3667-c25a-4c3c-8ba2-5f5d6910e339>`_, or embed the charts in your pages or blogs.
+
+   .. TODO: 
+      Max, can you embed the "confirmed cases" map 
+      and "confirmed cases growth" chart from your dashboard?
```

Comment:
@MaBeuLux88 is this possible? Or should I just take screenshots of them?

Reply:
I will include some charts.